test/testSetup.ts

```diff
@@ -0,0 +1,337 @@
+import { config } from 'dotenv';
+import request from 'supertest';
+import { AlkemioClient } from '@alkemio/client-lib';
+import { TestUser } from './utils/token.helper';
+import { getMails } from './utils/mailslurper.rest.requests';
+import { userData } from './utils/common-params';
+
+config({ path: '.env' });
+
+function delay(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+const kratosDomain =
+  'http://localhost:3000/identity/ory/kratos/public/self-service';
```

Comment:
this should be configurable, otherwise, you can't run the tests against other environments than local